.travis.yml

```diff
@@ -19,6 +19,7 @@ branches:
 
 addons:
   apt:
+    update: true
```

Comment:
Turns out you don't need this - it was just an availability thing.

Reply:
Ok